build.fsx

```diff
@@ -0,0 +1,22 @@
+#r @"packages\Build\Fake\tools\FakeLib.dll"
+open Fake
+let target = getBuildParam "target"
+let configuration = getBuildParamOrDefault "configuration" "Release"
+let signOutput = hasBuildParam "signOutput"
+
+Target "Build" <| fun _ ->
+    DotNetCli.Build <| fun p ->
+        { p with
+            Configuration = configuration
+            Project = "EFCore.FSharp.sln" }
```

Comment:
Flow `signOutput`? (This lets me sign the builds before releasing to NuGet.org)

Reply:
Let me see how to do that :-)

Reply:
How should this flow into MSBuild? `/p:SignOutput=True` or something else?

Reply:
@bricelam ok I just pushed some extra bits to the FAKE script, this should now be working (hopefully!).